runtime/lua/vim/lsp/util.lua

```diff
@@ -438,6 +438,7 @@ function M.jump_to_location(location)
 
   --- Jump to new location
   api.nvim_set_current_buf(bufnr)
+  api.nvim_buf_set_option(bufnr, 'buflisted', true)
```

Comment:
Should I pass `0` instead of passing `bufnr`? Which is faster, passing the exact buffer or getting the current one?

Reply:
"0" is faster, but it doesn't make a difference as much as I care about here, so I think either one is fine.

https://github.com/neovim/neovim/blob/master/src/nvim/api/buffer.c#L929
https://github.com/neovim/neovim/blob/master/src/nvim/api/private/helpers.c#L625-L631
https://github.com/neovim/neovim/blob/master/src/nvim/api/private/handle.c#L16-L19

Reply:
Oh, I see... thanks and sorry for making you dig through source code, I'm still new to the Neovim codebase.